### PR TITLE
docs: add github actions badges to README

### PR DESCRIPTION
**Pull request by y-prudent**
**New badges:**
* colored python 3.8+ badge
* github actions badges for pylint, pytest [torch] and pytest [tensorflow]

Check [here](https://github.com/deel-ai/oodeel/tree/docs/fancy_README) to see the result.

### CODE REVIEW

**Review by paulnovello — CHANGES_REQUESTED** [review]
The linter is flake8, be careful. Otherwise perfect

File: .github/workflows/python-tests-tf.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: oodeel tests tensorflow
```
Comment (cofri) [inline .github/workflows/python-tests-tf.yml:1]: tox is a generic tool to run multiple checks and tests. It is not specific to unit testing. I think `pytest` or `unit tests` is a better name for this GH action.

File: .github/workflows/python-tests-torch.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: oodeel tests torch
```
Comment (cofri) [inline .github/workflows/python-tests-torch.yml:1]: Same here, I suggest a more relevant name, e.g. `pytest`

**Review by cofri — APPROVED** [review]

**Review by paulnovello — APPROVED** [review]
LGTM

**Pull request merged**